### PR TITLE
Providers v2.0.1

### PR DESCRIPTION
**Pull request by mrjvs**
changes:
 - fix febbox not showing every quality
 - fix discoverEmbeds not showing

### CODE REVIEW

File: src/runners/runner.ts [hunk 1]
```diff
@@ -116,20 +116,20 @@ export async function runAllProviders(list: ProviderList, ops: ProviderRunnerOpt
       };
     }
 
-    if (output.embeds.length > 0) {
+    // run embed scrapers on listed embeds
+    const sortedEmbeds = output.embeds;
+    sortedEmbeds.sort((a, b) => embedIds.indexOf(a.embedId) - embedIds.indexOf(b.embedId));
```
Comment (JipFr) [inline src/runners/runner.ts:121]: This is how I'd do it, if you no likey you can just resolve this thread. 

```suggestion
    const sortedEmbeds = output.embeds.sort((a, b) => embedIds.indexOf(a.embedId) - embedIds.indexOf(b.embedId));
```

**Review by JipFr — COMMENTED** [review]
Mostly LGTM

**Comment by mrjvs** [comment]
Technically im not supposed to commit directly to dev, but I can avert my eyes

**Review by JipFr — APPROVED** [review]
Good soup

**Pull request merged**